src/django_secret_sharing/views.py

```diff
@@ -64,3 +65,11 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["value"] = value
         return context
+
+
+class GeneratePasswordView(CreateSecretView):
+
+    def get_initial(self):
+        initial = super().get_initial()
+        initial["value"] = get_random_string(32)
```

Comment:
kunnen we hier een setting voor maken?
dus dat een third-party adhv django settings de lengte kan wijzigen (32 is best lang voor velen)